src/features/customerView/CustomerViewContainer.tsx

```diff
@@ -36,7 +36,11 @@ const CustomerViewContainer = () => {
   if (loading) return <LoadingSpinner isLoading={loading} />;
   if (!data?.profile || !data?.boatTypes)
     return (
-      <Notification labelText={t('common.notification.noData.label')}>
+      <Notification
+        size="large"
+        closeButtonLabelText={t('toast.closeToast') ?? ''}
```

Comment:
Should we use these as defaults for our Notification wrapper?

Reply:
It doesn't seem that we have a wrapper in use, I can create one.

Reply:
Well, I guess we don't need one. Let's stick with this.